daisy_workflows/build-publish/rhui/rhua_artifacts/rhua.patch

```diff
@@ -73,3 +73,16 @@ diff -Naur playbooks/rhua-provision.yml playbooks/rhua-provision.yml
  
      - name: Collect static browsable API and media files
        command: "{{ pulpcore_env }} pulpcore-manager collectstatic --noinput"
+@@ -402,6 +407,12 @@
+     - name: Set SELinux httpd_can_network_connect boolean and keep it persistent across reboots
+       command: /usr/sbin/setsebool -P httpd_can_network_connect on
+ 
++    - name: Allow drop-ins for NginX
++      lineinfile:
++        dest: "{{ template_prefix }}/nginx-pulp.conf"
++        insertafter: "http {"
++        line: "    include /etc/nginx/conf.d/*.conf;"
++
+     - name: Provide NginX Pulp specific configuration
+       template:
+         src: "{{ template_prefix }}/nginx-pulp.conf"
```

Comment:
```suggestion

--- templates/nginx-pulp.conf	2022-01-01 23:56:33.757374995 +0000
+++ templates/nginx-pulp.conf	2022-01-01 23:56:47.768584643 +0000
@@ -10,6 +10,7 @@
 }
 
 http {
+    include /etc/nginx/conf.d/*.conf
     include mime.types;
     # fallback in case we can't determine a type
     default_type application/octet-stream;
```